### ISSUE TITLE
How to use batch processing

### ISSUE THREAD

**Issue by Jandown** [hkchengrex/MMAudio#25]
Great tool
May I ask how to use the newly added batch processing function? I couldn't find any relevant instructions. Can you provide an answer? Thank you

**Comment by hkchengrex** [comment]
Does this help? https://github.com/hkchengrex/MMAudio/blob/main/docs/EVAL.md

**Comment by exedesign** [comment]
I don't think this is what we are looking for, and even if it is, it's too complicated.

**Comment by hkchengrex** [comment]
I'm not sure what you are looking for.

`batch_eval.py` would still be a good starting point for writing your batch evaluation script. It calls the `generate` function which is also used in `demo.py` but in a batch setting and includes a few important optimizations for efficient inference.

**Comment by exedesign** [comment]
> Ne aradığınızı bilmiyorum.
> 
> `batch_eval.py`toplu değerlendirme betiğinizi yazmak için yine de iyi bir başlangıç ​​noktası olacaktır. Toplu ayarda `generate`kullanılan `demo.py`ve verimli çıkarım için birkaç önemli optimizasyon içeren işlevi çağırır.

Actually, I wrote a script called demo.py. It works by processing videos in a folder in bulk. I just had a hard time understanding this method. Please forgive me if I sounded rude.

**Comment by hkchengrex** [comment]
No worries. Glad that it worked.

**Comment by TheMindExpansionNetwork** [comment]
> > Ne aradığınızı bilmiyorum.
> > `batch_eval.py`toplu değerlendirme betiğinizi yazmak için yine de iyi bir başlangıç ​​noktası olacaktır. Toplu ayarda `generate`kullanılan `demo.py`ve verimli çıkarım için birkaç önemli optimizasyon içeren işlevi çağırır.
> 
> Actually, I wrote a different batch script using demo.py. It works by processing videos in a folder in bulk. I just had a hard time understanding this method. Please forgive me if I sounded rude.

do you have that script by chance cuz I am with you so simple yet stressful

used pyscene to take video and seperate out but have a bulk folder

just want to have it have folder then detect the secs then uses that for create auto short movie stuff

**Comment by exedesign** [comment]
> > > Ne istediğimi bilmiyorum. 
> > > `batch_eval.py`Toplu değerlendirme betiğinizi yazmak için yine de iyi bir başlangıç ​​​​noktası olacaktır. Toplu ayarda `generate`kullanılan `demo.py`ve verimli çıkarım için birkaç önemli değişiklik içeren işlevi çağırır.
> > 
> > 
> > Aslında, demo.py kullanarak farklı bir toplu iş betiği yazdım. Bir klasördeki videoları toplu olarak işleyerek çalışıyor. Bu yöntemi anlamakta zorluk çektim. Kaba görünüyorsam lütfen beni affedin.
> 
> bu senaryoyu şans eseri mi buldun çünkü seninleyim çok basit ama stresli
> 
> videoyu çekmek ve ayırmak için pyscene kullandım ama toplu bir klasörüm var
> 
> sadece klasörü olmasını istiyorum sonra saniyeleri algılayıp sonra onu otomatik kısa film oluşturmak için kullanmak istiyorum

![image](https://github.com/user-attachments/assets/cea2878a-fae4-46a9-9ec4-e775e45836e1)
is working successfully.
